config/environments/staging.rb

```diff
@@ -0,0 +1,89 @@
+Rails.application.configure do
```

Comment:
Regading configuration, it is key that staging remains as close to production as possible, now and in the future.

Therefore copy the production_or_staging.rb file/pattern from kuina.